tests/foreman/cli/test_role.py

```diff
@@ -134,6 +134,74 @@ def test_positive_update_name(self):
                 role = Role.info({'id': role['id']})
                 self.assertEqual(role['name'], new_name)
 
+    @tier1
+    def test_positive_list_filters_by_id(self):
```

Comment:
This test is more filter-centric and checks that correct filter id was listed and not permissions. But this case may be redundant indeed.